src/components/tutorial/Slides.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { FILLED_COLOR, WHITE } from "../../graphics/theme/ColorScheme";
+import { BOARD_DIMENSIONS, COLUMN_WIDTH } from "../../graphics/theme/Layout";
+import { container } from "../BaseStyles";
+import { interpolateColor } from "../../graphics/utils/Colors";
+import { linInt } from "../../graphics/utils/Functions";
+
```

Comment:
This code does have a very different layout from the others, best to leave a note that this is I'm trying to treat this with a Static HTML vibe since almost no interaction is needed